tests/models/test_petrinet.py

```diff
@@ -0,0 +1,77 @@
+
+import string
+import tempfile
+import unittest
+
+from pmkoalas.models.pnfrag import *
+from logging import *
+
+
+expectedXML = '''<pnml>
+  <net type="('http://www.pnml.org/version-2009/grammar/pnmlcoremodel',)" id="dotTest">
+    <page id="page1">
+      <place id="4">
+        <name>
+          <text>Student</text>
+        </name>
+      </place>
+      <place id="3">
+        <name>
+          <text>Sweep</text>
+        </name>
+      </place>
+      <place id="1">
+        <name>
+          <text>I</text>
+        </name>
+      </place>
+      <transition id="2">
+        <name>
+          <text>tau</text>
+        </name>
+        <toolspecific tool="StochasticPetriNet" version="0.2" invisible="True" priority="1" weight="2.0" distributionType="IMMEDIATE" />
+      </transition>
+      <arc source="2" target="4" id="1" />
+      <arc source="1" target="2" id="2" />
+      <arc source="2" target="3" id="3" />
+    </page>
+  </net>
+</pnml>
+'''
+
```

Comment:
This seems light for a major structure. Perhaps we should have a few more functional checks for this structure?

Reply:
Requires changing the export implementation to be strictly deterministic by implementing sort comparators for places, transitions and arcs. Perhaps desirable, but I deliberately cut from the scope.

Reply:
If you are talking about PN itself, the coverage provided by pnfrag is already substantial, as the comment notes.